### DISCUSSION TITLE
Issues being closed means past issue history effectively lost

### DISCUSSION THREAD

**Discussion by kevinlang** [alpinejs/alpine#1453] [4. General]
Per #1423 , a lot of the issue history is now lost, which makes it significantly harder to debug issues one comes across as the past history is gone.

For example, I'm looking into an issue related to "@click is not a valid attribute", and searched that on google and found this issue:
https://github.com/alpinejs/alpine/issues/396 . However, once I click through all I get is a 404 :sob: . I can view the Google cache, but that won't last forever...

**Comment by calebporzio** [comment]
Ah, very true @kevinlang - I didn't consider that. I suppose then that this is a good time to make the switch as Alpine now has a different codebase, but I totally get it - not ideal at all.

Do you have any ideas?

**Reply by calebporzio** [reply]
I just re-enabled issues and it looks like all the history is preserved.

Wondering if the best thing to do here is leave them enabled, add an issue template the points people to discussions, and maybe even create a bot that auto-closes all new issues.

**Reply by ariaieboy** [reply]
i dont know that its late or not but one approach is to seperate alpine v2 from v3 codebase like what happened in vuejs. and keep them in 2 different repo.

**Reply by kevinlang** [reply]
Honestly it kinda seems like a bug on GitHub! They should allow an option to lock but keep the issues visible. That could even involve removing the issues tab from the repository but still preserving deep links for crawling purposes.

EDIT:

I looked into this a bit further. One other possible solution is to make an `alpinejs/alpine2` repository, transfer all of the issues on this repository over to it, and then archive that. That would keep this repository having the stars/etc. I believe after the transfer you can re-disable issues on this repository. It may be worth trying this out on an example repository first to confirm the behavior is as expected though. 

https://docs.github.com/en/issues/tracking-your-work-with-issues/managing-issues/transferring-an-issue-to-another-repository

**Reply by KevinBatdorf** [reply]
> transfer all of the issues on this repository

I just tried this out and GH will 302 redirect those URLs as well, so that seems like it would work.

**Reply by HugoDF** [reply]
> > transfer all of the issues on this repository
> 
> I just tried this out and GH will 302 redirect those URLs as well, so that seems like it would work. 

Will it still 302 when you create an `alpine` repo? I wouldn't think it would
